3_javascript/javascript1/week1/javascript.js

```diff
@@ -0,0 +1,120 @@
+console.log('------------- Age - ify(A future age calculator)-------------');
+
+let yearOfBirth = 1976;
+let yearFuture = 2076;
+let age = yearFuture - yearOfBirth;
+
+console.log('You will be ' + age + ' years old in ' + yearFuture);
+console.log('\n');
+
+
+console.log('----------- Goodboy-Oldboy (A dog age calculator) -----------');
+let dogYearOfBirth = 2018;
+let dogYearFuture = 2036;
+let dogYear;
+
+shouldShowResultInDogYears: Boolean;
+
+let choice = prompt("Do you wnat to know your dog age based on Dog years or Human years calculation? \n Type the word dog or human please");
+
+if (choice == 'dog') {
+    let shouldShowResultInDogYears = (choice === 'true');
+    console.log('You have chosen the word ' + choice);
+    console.log('Your dog age calculation based on dog years: ');
+    dogYear = (yearFuture - dogYearOfBirth) * 7;
+    console.log('Your dog will be ' + dogYear + ' dog years old in ' + dogYearFuture);
+} else if (choice == 'human') {
+    console.log('You have chosen the word ' + choice);
+    console.log('Your dog age calculation based on human years: ');
+    dogYear = yearFuture - dogYearOfBirth;
+    console.log('Your dog will be ' + dogYear + ' human years old in ' + dogYearFuture);
+} else {
+    console.log('You have not entered dog neither human. \n Please run the page again and enter one of them to perform the calculations');
+}
```

Comment:
I would suggest to change this block a little to make it more simple and readable. As you can see I'm again using template literals in order to insert variable values into the strings. And I'm also using strict equality(triple equal) operator instead of double equal one in order to be sure that both types and  values are the same.
```suggestion
let shouldShowResultInDogYears;

let choice = prompt(
  `Do you want to know your dog's age based on dog years or human years calculation? \n Type the word "dog" or "human" please.`
);

if (choice === "dog") {
  shouldShowResultInDogYears = true;
  console.log(`You have chosen the word "${choice}".`);
} else if (choice === "human") {
  shouldShowResultInDogYears = false;
  console.log(`You have chosen the word "${choice}".`);
} else {
  alert(
    "You have not entered dog neither human. \n Please run the page again and enter one of them to perform the calculations."
  );
  return;
}

if (shouldShowResultInDogYears) {
  console.log(`Your dog's age calculation based on dog years:`);
  dogYear = (dogYearFuture - dogYearOfBirth) * 7;
  console.log(
    `Your dog will be ${dogYear} dog years old in ${dogYearFuture}.`
  );
} else {
  console.log(`Your dog's age calculation based on human years:`);
  dogYear = dogYearFuture - dogYearOfBirth;
  console.log(`Your dog will be ${dogYear} human years old in ${dogYearFuture}.`);
}
```

Reply:
Hi @margarit,
I have applied the suggestions. Thank you :)